back/engines/commercial/user_custom_fields/spec/acceptance/r_scores_spec.rb

```diff
@@ -11,94 +11,112 @@
           Project ID. Only participants of this project will be considered to compute the score.
     DESC
 
-    let(:ref_distribution) { create(:categorical_distribution) }
-    let(:custom_field) { ref_distribution.custom_field }
-    let(:custom_field_id) { custom_field.id }
+    let(:custom_field_id) { 'whatever' }
```

Comment:
Is this correct? Looks a bit like a placeholder?

Reply:
Yep, that's intentional. This would still work without setting the `let`-helper explicitly, but I wanted to make clear that we don't care about the actual value at this level. Will add a comment for this.